### PR TITLE
Simplification of the _s_categorized_blog() return

### PR DESCRIPTION
**Pull request by alvisir**
Not a big change but a recommended simplification.

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -99,13 +99,7 @@ function _s_categorized_blog() {
...
 	}
 
-	if ( $all_the_cool_cats > 1 ) {
-		// This blog has more than 1 category so _s_categorized_blog should return true.
-		return true;
-	} else {
-		// This blog has only 1 category so _s_categorized_blog should return false.
-		return false;
-	}
+	return ( $all_the_cool_cats > 1 );
```
Comment (jrfnl) [inline inc/template-tags.php]: Maybe summarize the documentation which is now being removed and add it above the line ?

Comment (alvisir) [inline inc/template-tags.php]: You're right. Just updated the commit

File: inc/template-tags.php [hunk 2]
```diff
@@ -98,14 +98,9 @@ function _s_categorized_blog() {
...
-
-	if ( $all_the_cool_cats > 1 ) {
-		// This blog has more than 1 category so _s_categorized_blog should return true.
-		return true;
-	} else {
-		// This blog has only 1 category so _s_categorized_blog should return false.
-		return false;
-	}
+	
+	// If this blog has more than 1 category the function return true, otherwise false.
```
Comment (jrfnl) [inline inc/template-tags.php]: `return` => `returns`

Comment (alvisir) [inline inc/template-tags.php]: Ouch! Thanks ;)

File: inc/template-tags.php [hunk 3]
```diff
@@ -98,14 +98,9 @@ function _s_categorized_blog() {
...
-	if ( $all_the_cool_cats > 1 ) {
-		// This blog has more than 1 category so _s_categorized_blog should return true.
-		return true;
-	} else {
-		// This blog has only 1 category so _s_categorized_blog should return false.
-		return false;
-	}
+	
+	// If this blog has more than 1 category the function returns true, otherwise false.
+	return ( $all_the_cool_cats > 1 );
```
Comment (mtomas7) [inline inc/template-tags.php]: If we would also implement https://github.com/Automattic/_s/pull/1078/files

`return ( $all_the_cool_cats > 1 || is_preview() );`

**Comment by davidakennedy** [comment]
We've removed this function as a result of #1131. Closing this out. Thanks for the work here @alvisir!

**Pull request closed without merging**